src/scripts/sync_subclassof.py

```diff
@@ -410,6 +413,13 @@ def sync_subclassof(
     df3_obs.to_csv(outpath_added_obsolete, sep='\t', index=False)
     df3 = pd.concat([pd.DataFrame(subheader), df3])
     df3.to_csv(outpath_added, sep='\t', index=False)
+    # - issue: Mondo IDs subClassed to themselves. See: https://github.com/monarch-initiative/mondo-ingest/issues/400
+    df3_self_parentage = df3[df3['subject_mondo_id'] == df3['object_mondo_id']]
+    sp_err = f'Classes of self-parentage found for source {source_name}. This is a low urgency warning, as these are ' \
+             f'with almost 100% certainty caused by legal Mondo proxy merges.'
```

Comment:
Warning message
- [ ] @matentzn @twhetzel review and suggest any changes. I wasn't sure how to phrase.

Reply:
The phrasing is not wrong, no strong opinions